packages/waku/src/router/fs-router.ts

```diff
@@ -77,6 +77,19 @@ export function fsRouter(
             render: 'static',
             ...config,
           });
+        } else if (
+          process.env.npm_lifecycle_script?.includes('static') &&
```

Comment:
I would like to avoid using `process`.
L18 checks if it's PRD, but in general we make all code (except for dev/build specific files) free from Node.js. This is still confusing and I hope to organize the code structure better.

Reply:
Makes sense. 

Should there be some concept of a "build context" where we pass a state object in with all of the state for a given build. 

Here that would mean passing the build mode (static or dynamic) but I imagine other pieces of state would be helpful to pass around to many of the builder functions.

If not this, how else would you like to communicate to fs-router what the build type is?

Reply:
At this point, I don't think it's possible. It's due to my current design which I find limited, and for now lots of hacks around "build config".
I would like to revamp things around it after finalizing `v0.21.0`. Can consider about "build context" too then.

So, for now let's wait for it and mitigate the issue with readme, please?